src/backend/cuda/platform.cpp

```diff
@@ -469,8 +482,59 @@ SparseHandle sparseHandle() {
     return cusparseHandles[id].get()->get();
 }
 
+/*
+ * Map giving the minimum device driver needed in order to run a given version of CUDA
+ * https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html
+ * Map linking cuda version to the minimum driver versions for both Linux/Mac and Windows
+ */
+static const std::map<float, std::pair<double, double> > CudaToKernelVersionMap = {
```

Comment:
`static const` -> `constexpr`

Reply:
does nt compile under gcc 5

Reply:
sure ? because we have `constexpr` in various other locations in the code, in fact in CUDA backend itself. Check this one out.
https://github.com/arrayfire/arrayfire/blob/master/src/backend/cuda/Array.cpp